### DISCUSSION TITLE
Sorting / Filtering Not Working

### DISCUSSION THREAD

**Discussion by davhad** [AntlerVC/firetable#1315] [Support / Q&A] [answered]
Please see this jam recording: https://jam.dev/c/ccdb58ce-171a-4fcb-92d4-d1b3f25e8cf6

**Comment by BeeBombshell** [comment, marked as answer]
Hey @davhad, yes. We've come across some issues with the Sorting and Filtering features. An interesting thing to note, these issues do not occur on every table, rather it is something specific to the table schema. We're trying to investigate this further. Thanks for bringing this up!

**Comment by bofeiw** [comment]
Created an issue with details for it - https://github.com/rowyio/rowy/issues/1527
